app/channels/room_channel.rb

```diff
@@ -1,24 +1,16 @@
 class RoomChannel < ApplicationCable::Channel
-  # identified_by :room_id
-  # stream_for room
+    def subscribed
+        stream_for room
+    end
 
-  def subscribed
-    # stream_from "some_channel"
-    # stream_from User.where('room_id = ?', params[:room_id])
-    stream_for room
-  end
-  
-  def connect
-    self.room_id = User.find_by(id: cookies.encrypted[:user_id])[:room_id]
-  end
```

Comment:
remember this part if our broadcast stops working for some reason.

Reply:
good thing we have source control to look at previous versions of the code